### PR TITLE
Fix: Fortigate datetime issue

### PR DESCRIPTION
**Pull request by vg-svitla**
Closes [300](https://github.com/SekoiaLab/integration/issues/300)

### CODE REVIEW

File: Fortinet/fortigate/ingest/parser.yml [hunk 1]
```diff
@@ -52,31 +52,31 @@ pipeline:
...
   - name: parsed_date
-    filter: '{{parsed_event.message.get("eventtime") != None}}'
+    filter: '{{parsed_event.message.get("timestamp") != None}}'
     external:
       name: date.parse
       properties:
-        input_field: "{{parsed_event.message.eventtime }}"
+        input_field: "{{parsed_event.message.timestamp }}"
         output_field: date
         timezone: "UTC"
```
Comment (squioc) [inline Fortinet/fortigate/ingest/parser.yml:61]: According to some observations:
The timestamp is the local one from the machine.
The eventtime is a timestamp, in nanosecond, in the UTC timezone

So, for the parsing of the timestamp, I suggest using `{{parsed_event.message.tz}}` as the timezone.

**Review by squioc — CHANGES_REQUESTED** [review]

**Comment by rombernier** [comment]
Hello @vg-svitla, can you also add this update in the PR ? https://github.com/SekoiaLab/integration/issues/320

**Comment by rombernier** [comment]
Hello, any ETA on this fix ? Thanks

**Comment by squioc** [comment]
@rombernier No yet. I'm trying to qualify the Fortigate fields with date in order to not introduce a regression in the format.

**Pull request merged**